packages/mdc-ripple/util.js

```diff
@@ -15,13 +15,21 @@
  */
 
 /** @private {boolean|undefined} */
+let supportsCssVariables_;
```

Comment:
I was under the impression we don't include docs in our comments for the closure compiler, and this variable is very similar in function to the one immediately below it which is also not documented.  Should I be documenting both?